### PR TITLE
Small improvements in dev VM scripts and setup

### PR DESCRIPTION
**Pull request by aanm**
read per commit basis

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8403)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

File: Vagrantfile [hunk 1]
```diff
@@ -124,6 +124,7 @@ Vagrant.configure(2) do |config|
     config.vm.provision "bootstrap", type: "shell", inline: $bootstrap
     config.vm.provision "build", type: "shell", run: "always", privileged: false, inline: $build
     config.vm.provision "install", type: "shell", run: "always", privileged: false, inline: $install
+    config.vm.box_check_update = false
```
Comment (ianvernon) [inline Vagrantfile]: I disagree with this being the default behavior - by getting the latest and greatest by default, we get improvements / newer VM versions, etc. I'd prefer if you wrap this with an environment variable, `CHECK_UPDATES=false` where by default, it is `true`.

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by aanm** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**